sources/libs/stable_curve.move

```diff
@@ -147,26 +147,21 @@ module liquidswap::stable_curve {
     }
 
     /// Trying to find suitable `y` value.
-    /// * `x0` - total reserve x (include `coin_in`) with transformed decimals.
+    /// * `x0` - total reserve x (include `coin_in`/exclude `coin_out`) with transformed decimals.
     /// * `xy` - lp value (see `lp_value` func).
-    /// * `y` - reserves out with transformed decimals.
+    /// * `y` - reserves y with transformed decimals.
     fun get_y(x0: U256, xy: U256, y: U256): U256 {
         let i = 0;
 
-        let one_u256 = u256::from_u128(1);
-
         while (i < 255) {
             let k = f(x0, y);
 
             let _dy = u256::zero();
             let cmp = u256::compare(&k, &xy);
             if (cmp == 1) {
-                _dy = u256::add(
-                    u256::div(
-                        u256::sub(xy, k),
-                        d(x0, y),
-                    ),
-                    one_u256    // Round up
```

Comment:
There was no need to add one, because it will be added [here](https://github.com/pontem-network/liquidswap/blob/main/sources/swap/router.move#L440)